src/launch/config.c

```diff
@@ -1259,8 +1259,14 @@ static int config_parser_include(ConfigParser *parser, ConfigRoot *root, ConfigN
 
         r = open(node->include.file->path, O_RDONLY | O_CLOEXEC | O_NOCTTY);
         if (r < 0) {
-                if (errno == ENOENT || errno == ENOTDIR)
-                        return node->include.ignore_missing ? 0 : CONFIG_E_INVALID;
+                if (errno == ENOENT || errno == ENOTDIR) {
+                        if (node->include.ignore_missing)
+                                return 0;
+
+                        CONFIG_ERR(&parser->state, "Invalid configuration file",
```

Comment:
Shouldn't the message say "Missing" rather than "Invalid"?

Reply:
Right. Fixed.